Week 2 - Hello World Assignment

larawillson opened the pull request:
Week 2 - Hello World Assignment - Lara Willson - COP1000 - Spring 2016

halfaro1 commented:
@larawillson great work! I have one question: how come the code is enveloped in a `while True` loop?

larawillson commented:
@halfaro1 I was trying to make the program prompt the user with the question until they answered using one of the three options. Is there a better way to do this? It was late, and I was definitely struggling with that.

I have gone ahead and removed the loop and added an exit loop at the end of the program (just so I could get the practice with looping :))

halfaro1 commented:
@larawillson great work, I love the enthusiasm :+1:
